Apromore-Custom-Plugins/Process-Discoverer-Portal-Plugin/src/main/java/org/apromore/plugin/portal/processdiscoverer/PDDesktopCleaner.java

```diff
@@ -39,13 +39,13 @@
 
     @Override
     public void cleanup(Desktop desktop) throws Exception {
-        LOGGER.info("PD cleanup starts for desktopID = " + desktop.getId());
+        LOGGER.debug("PD cleanup starts for desktopID = " + desktop.getId());
```

Comment:
This should be INFO level as it marks the end of the plugin execution.

Reply:
As above, this is not helpful for the system administrator to see.

Reply:
Reinstated to INFO level.